lib/reqTemplate.js

```diff
@@ -157,6 +220,19 @@ function Template(spec) {
 }
 
 /**
+ * Adds common functions to the context.func object,
+ * so that they could be used in templates as rm.func.funcName
+ *
+ * @param {object} context a request context to put common functions to
+ */
+function appendCommonOperators(context) {
```

Comment:
Normally the model is reserved for data. It might be cleaner to use `rc.g` here to define a global encodeURIComponent function at compile time. That should also make it more likely that this function is inlined by the JIT.